api/base/exceptions.py

```diff
@@ -19,10 +21,16 @@ def json_api_exception_handler(exc, context):
         message = response.data
 
         if isinstance(exc, JSONAPIException):
+            # TODO: We're now stripping html from exception detail to protect
```

Comment:
I'm not sure that this is a good idea. See my and @abought 's comments here: https://github.com/CenterForOpenScience/osf.io/pull/4246#discussion_r42010701 .